GVFS/GVFS.UnitTests/Common/GitVersionTests.cs

```diff
@@ -10,9 +11,9 @@ public class GitVersionTests
         [TestCase]
         public void TryParseInstallerName()
         {
-            this.ParseAndValidateInstallerVersion("Git-1.2.3.gvfs.4.5.gb16030b-64-bit.exe");
-            this.ParseAndValidateInstallerVersion("git-1.2.3.gvfs.4.5.gb16030b-64-bit.exe");
-            this.ParseAndValidateInstallerVersion("Git-1.2.3.gvfs.4.5.gb16030b-64-bit.EXE");
+            this.ParseAndValidateInstallerVersion("Git-1.2.3.gvfs.4.5.gb16030b-64-bit" + GVFSPlatform.Instance.Constants.InstallerExtension);
+            this.ParseAndValidateInstallerVersion("git-1.2.3.gvfs.4.5.gb16030b-64-bit" + GVFSPlatform.Instance.Constants.InstallerExtension);
+            this.ParseAndValidateInstallerVersion("Git-1.2.3.gvfs.4.5.gb16030b-64-bit" + GVFSPlatform.Instance.Constants.InstallerExtension);

```

Comment:
Does the third line in this test actually test anything now that we're using a constant?

Reply:
Yes - it tests the TryParseInstallerName() in GitVersion.cs. That method has been updated to accept a Platform dependent InstallerExtension.